app/bin/run

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# Terminate the running service
+screen -X -S DevopsMonitorBackend quit
+
+# Go to the project directory and pull changes
+cd /root/devops-monitor-backend
+git fetch origin master
```

Comment:
why not just `git pull` ?

Reply:
`git pull` will not reset changes that are made to the repository - e.g. i made changes in production env and git pull will not reset those

Reply:
that's not a good approach to make changes on production code.